project/interactivemap/settings.py

```diff
@@ -129,7 +134,18 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 
-# Graphene
-GRAPHENE = {
-    'SCHEMA': 'interactivemap.schema.schema' # Where your Graphene schema lives
+REST_FRAMEWORK = {
+    # Use Django's standard `django.contrib.auth` permissions,
+    # or allow read-only access for unauthenticated users.
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.IsAuthenticated',
+    ],
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.TokenAuthentication',
+    )
 }
+
+CORS_ORIGIN_WHITELIST = (
+    'localhost:3000',
+    'frontend-*.now.sh',
```

Comment:
I updated the name on the frontend, it's `interactivemap-frontend-*.now.sh` now.

Reply:
Noted